# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/ServiceWorkerContainerParent.h
# Commit: 244df402d6eb
# Full Hash: 244df402d6ebc15d6845c4144245511021408835
# Author: Andrew Sutherland <asutherland@asutherland.org>
# Date: 2024-10-24 09:44:34
# Regressor Bug: 1131324
# File Overlap Count: 1
# Description:
#   Bug 1131324 - Expose ServiceWorkerContainer on WorkerNavigator. r=dom-worker-reviewers,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213725
# ==============================================================================

diff -r 57f70ba5f27e -r 244df402d6eb dom/serviceworkers/ServiceWorkerContainerParent.h
--- a/dom/serviceworkers/ServiceWorkerContainerParent.h	Thu Oct 24 03:02:39 2024 +0000
+++ b/dom/serviceworkers/ServiceWorkerContainerParent.h	Thu Oct 24 03:02:40 2024 +0000
@@ -23,8 +23,6 @@
   // PServiceWorkerContainerParent
   void ActorDestroy(ActorDestroyReason aReason) override;
 
-  mozilla::ipc::IPCResult RecvTeardown() override;
-
   mozilla::ipc::IPCResult RecvRegister(
       const IPCClientInfo& aClientInfo, const nsACString& aScopeURL,
       const nsACString& aScriptURL,